### PR TITLE
moveKnight method commented out

### PR DESCRIPTION
**Pull request by ccubukcu**
I have commented out the moveKnight method.

### CODE REVIEW

File: ChessBoard.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ def movePawn(self, fromPos, toPos):
 
         self._fifty = 0
         return True
-
```
Comment (nsomar) [inline ChessBoard.py:509]: Why did you remove this line?

Comment (ccubukcu) [inline ChessBoard.py:509]: I didn't remove. I re-purposed.

Comment (nsomar) [inline ChessBoard.py:509]: Ok, i see, please rebase to the newest master before merging.

Comment (ccubukcu) [inline ChessBoard.py:509]: I have rebased to the newest master.
